src/DotNet.Status.Web/appsettings.json

```diff
@@ -13,6 +13,27 @@
   "ApplicationInsights": {
     "InstrumentationKey": "[vault(app-insights-instrumentation-key)]"
   },
+  "BuildMonitor": {
+    "Monitor": {
+      "BaseUrl": "https://dev.azure.com",
+      "Organization": "dnceng",
+      "MaxParallelRequests": 10,
+      "AccessToken": "[vault(dn-bot-dnceng-build-rw-code-rw-release-rw)]",
+      "Builds": [
+        {
+          "Project": "internal",
```

Comment:
@Chrisboh Just wanted to call out this one little section here.  I wanted to turn it on for one repo first and make sure it's behaving in the wild, and wanted to make sure you were on board with getting this particular repo and branches assigned to you. :-)